src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -170,21 +191,18 @@ function configureTile(tile, layer, metadata, parent) {
     tile.layer = layer.id;
 
     // parse metadata
-    tile.transform = metadata.transform ? (new THREE.Matrix4()).fromArray(metadata.transform) : new THREE.Matrix4();
-    tile.applyMatrix(tile.transform);
+    if (metadata.transform) {
+        tile.applyMatrix(metadata.transform);
+    }
     tile.geometricError = metadata.geometricError;
     tile.tileId = metadata.tileId;
     if (metadata.refine) {
         tile.additiveRefinement = (metadata.refine.toUpperCase() === 'ADD');
     } else {
         tile.additiveRefinement = parent ? (parent.additiveRefinement) : false;
     }
-    tile.parentFromLocalTransform = tile.transform;
-    tile.worldFromLocalTransform = new THREE.Matrix4().multiplyMatrices(parent ? parent.worldFromLocalTransform : new THREE.Matrix4(), tile.parentFromLocalTransform);
-    const m = new THREE.Matrix4();
-    m.getInverse(tile.worldFromLocalTransform);
-    tile.viewerRequestVolume = metadata.viewerRequestVolume ? getBox(metadata.viewerRequestVolume, m) : undefined;
-    tile.boundingVolume = getBox(metadata.boundingVolume, m);
+    tile.viewerRequestVolume = metadata.viewerRequestVolume;
+    tile.boundingVolume = metadata.boundingVolume;
```

Comment:
Can you remove these? It is confusing to have the same information stored at two different places. I'd suggest only keeping a link to the metadata.

Reply:
Both prperties are used often, so removing them *could* have some perf impact.
I'd prefer to not remove them for now.

Reply:
That not exactly what I meant. My point was there should be a clearer separation between the metadata part of the object (bounding volume, etc.) and Object 3D part (transformation). Presently, both are mixed and sometimes we call culling on the metadata part, sometimes on the Object 3D part.